### ISSUE TITLE
No 2022 CBSA in Tiger database leads to error

### ISSUE THREAD

**Issue by camille-s** [walkerke/tigris#184]
Hi, I don't work too often with CBSAs so I'm not sure if this is a known issue already. The Tiger database doesn't have 2022 CBSAs---they're not listed in the PDF of geographies for the year ([CB](https://www2.census.gov/geo/tiger/GENZ2022/2022_file_name_def.pdf) or [TL](https://www2.census.gov/geo/tiger/TIGER2022/2022_TL_Shapefiles_File_Name_Definitions.pdf))---so calling `tigris::core_based_statistical_areas` letting the year default to 2022 yields an error that the download fails. 2021 and 2023 are in the database, so I'm not sure why 2022 is not.

**Comment by walkerke** [comment]
Good catch - I don't know why they omitted CBSAs for 2022.  I'll patch tigris to throw a descriptive error when 2022 is requested.
